### ISSUE TITLE
Can't find a tag without text

### ISSUE THREAD

**Issue by mihsan96** [nephi-dev/rxml#2]
Cant find a tag in list of children when his text is empty and a tag hasnt a pair.

```python
from rxml import read_file

root_read = read_file('xml/test.xml', 'tag')

print(root_read.children[0])
```

```
Name: class
Children:
  Name: test1
  Text: true

  Name: test3
  Text: true
```

```xml
<?xml version="1.0" encoding="utf-8"?>
<tag>
    <class>
        <test1>true</test1>
        <test2 resource="something"/>
        <test3>true</test3>
    </class>
</tag>
```

**Comment by nephi-dev** [comment]
tks for the report, i will try to fix this as soon as possible

**Comment by nephi-dev** [comment]
fixed on the [#2.1.1](https://github.com/nephi-dev/rxml/releases/tag/2.1.1) release
